lib/colorado_lottery.rb

```diff
@@ -0,0 +1,53 @@
+class ColoradoLottery
+  attr_reader :registered_contestants, :winners, :current_contestants
+
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(contestant, game)
+    contestant.age >= 18 && contestant.game_interests.include?(game.name)
+  end
+
+  def out_of_state_only_for_national(contestant, game)
```

Comment:
I like the use of a helper method here, but the naming is not as clear as it could be.